client/src/components/StreamField/StreamField.scss

```diff
@@ -15,5 +15,4 @@
 .c-sf-help .help {
   // StreamField block help text is above the blocks rather than below.
   margin-top: 0;
-  margin-bottom: theme('spacing.6');
```

Comment:
@TARishabh  this kind of helps but does not really fix the problem.

I would recommend we only 'reduce' this value, maybe to spacing.4 or spacing.3

Instead we should try to find a way to make the list block items have a margin under them I think.

Maybe if you adjust this file
https://github.com/wagtail/wagtail/blob/839193ff3c9381e35e39cc12f199c414d2684526/client/src/components/StreamField/blocks/ListBlock.js#L99-L101

with something like this

```diff
--      <div class="${h(this.blockDef.meta.classname || '')}">
++      <div class="${['w-mb-6', h(this.blockDef.meta.classname || '')].filter(Boolean).join(' ')}">
```

Then with these two suggestions we get a nicer visual gap between the ListBlock items.


<img width="1800" alt="Screenshot 2024-11-30 at 8 53 35 AM" src="https://github.com/user-attachments/assets/d306ec40-2983-4a78-be1f-a81fc5d39c92">